app/public/js/profile.js

```diff
@@ -109,15 +109,19 @@ function updateRegionOptions()
 	var country_id = parseInt(getCountryElement().val());
 	var $home_region = $('#home_region');
 	$home_region.empty();
+	//Below statement enables the default region to be empty or no region
+	$home_region.append($('<option/>').text("-- Select Region --"));
 	regions.forEach(function(region) {
 		if ( region.country_id === country_id )
-			if ($home_region.data('value') === region.name)
 			{
-				$home_region.append($('<option />').val(region.name).attr('selected', 'selected').text(region.name));
-			}
-			else
-			{
-				$home_region.append($('<option/>').val(region.name).text(region.name));
+				if($home_region.data('value') === region.name)
```

Comment:
I tested this and it works well.  It behaves properly for setting the blank option.

It doesn't fix the popup bug but that can be fixed by a future pull request.